### DISCUSSION TITLE
redis-cluster 模式 实例连接地址怎么写

### DISCUSSION THREAD

**Discussion by cheekboy** [hhyo/Archery#1628] [General]
现在集群模式  有多个ip  1.1.1.1-1.1.1.6:6379   在后台添加实例的时候地址怎么写

**Comment by nick2wang** [comment]
填写集群内任意一个ip即可

**Comment by cheekboy** [comment]
<img width="528" alt="image" src="https://user-images.githubusercontent.com/16200728/173806557-bd83a3bf-7842-4e5a-96b1-78bc4d8085c0.png">
1.8.5版本  模式选择cluster  有这个报错

**Reply by chenglei96** [reply]
大佬，这个问题解决了嘛

**Reply by nick2wang** [reply]
> 大佬，这个问题解决了嘛

手动更新下redis模块
pip install redis==4.1.0

**Reply by morgan9527** [reply]
> > 大佬，这个问题解决了嘛
> 
> 手动更新下redis模块 pip install redis==4.1.0

但是这样django-q 1.3.9就报错了
ERROR: 
django-q 1.3.9 requires redis<4.0.0,>=3.5.3, but you have redis 4.1.0 which is incompatible.

**Comment by chenglei96** [comment]
是跟新我需要连接redis是嘛，不是archery内置的那个redis

Nick Wang ***@***.***> 于2023年5月4日周四 16:21写道：

**Reply by cheekboy** [reply]
不太清楚你的版本是多少哈，1.85的版本  redis.py 这个文件可以尝试把    result.error = str(e) 这一行注释掉看看。

**Reply by chenglei96** [reply]
还是不太行，仍然报错

**Comment by chenglei96** [comment]
好的感谢，我先试下


wuzhu ***@***.***> 于2023年5月5日周五 10:30写道：

**Comment by morgan9527** [comment]
这个问题现在有解决吗？ 只能支持单机，或者将集群中master节点填写上，然后选择单机模式 才能使用
